open_instruct/finetune.py

```diff
@@ -393,7 +394,7 @@ def main():
 
     # no default pad token for llama!
     # here we add all special tokens again, because the default ones are not in the special_tokens_map
-    if isinstance(tokenizer, LlamaTokenizer):
+    if isinstance(tokenizer, LlamaTokenizer) or isinstance(tokenizer, LlamaTokenizerFast):
```

Comment:
I had some annoyances with the way the fasttokenizer dealt with adding tokens in the past, although it's probably fixed now. Worth double checking the tokenization makes sense after adding the special characters.

Reply:
I tried using the fasttokenizer in both training and inference and it seems okay. Let's merge it for now!